src/java.base/share/classes/java/util/Properties.java

```diff
@@ -1515,7 +1515,7 @@ public synchronized Object computeIfAbsent(Object key,
 
     @Override
     public synchronized @Nullable Object compute(Object key,
-            BiFunction<? super Object, ?, ?> remappingFunction) {
+            BiFunction<? super Object, ? super Object, ?> remappingFunction) {
```

Comment:
```suggestion
            BiFunction<? super Object, ? super @Nullable Object, ?> remappingFunction) {
```

How about this? I actually tried `make jdk` this time, and it appears to work, unlike the original code. I think we do want the nullable type for the second input type, since the table will pass `null` if the key wasn't found.

Reply:
How about keeping the change without annotations for this PR, to just make the build compile?
Then please open a follow-up PR that discusses the right annotations for this API.

Reply:
That's fair: I think I must have written `? super @Nullable Object` when I was writing https://github.com/jspecify/jdk/pull/33 and then replaced it with `?` (thinking, mistakenly, that I'd originally written ``? extends @Nullable Object` instead). Anything that we do in this PR will be a change in meaning from what we have now, so we can negotiate that later.